components/src/core/hero/hero.component.scss

```diff
@@ -12,17 +12,17 @@
 
 .pxb-hero-content {
     display: flex;
-    flex: 1 1 0px;
+    flex: 1 1 0;
     flex-direction: column;
     align-items: center;
     justify-content: flex-start;
     font-size: 1rem;
     overflow: hidden;
-    padding: 16px 8px;
+    padding: 1rem 0.5rem;
```

Comment:
why 0.5rem for horizontal padding instead of 8px?

Reply:
No reason, this is updated now.